tetrateci/docs/fips.md

```diff
@@ -0,0 +1,105 @@
+## Introduction
+
+Google's BoringCrypto [module][1] is used for [FIPS-compliant Istio builds][2]. BoringCrypto is a core module of the
+BoringSSL library and has been tested by CMVP to be [FIPS validated][3]. Both the Istio control plane and data plane
+are built with these modules. The quickest way to get started with FIPS Istio is to use the
+[Tetrate Istio Distribution][4].
+
+## FIPS Build Process and Verification
+
+### Istio Control Plane FIPS Build Process
+
+This [doc](https://gokulchandrapr.medium.com/go-crypto-and-kubernetes-fips-140-2-fedramp-compliance-66d852ccccd2)
+provides a good introduction on what's necessary to build a FIPS compliant go binaries by using boring crypto.
+
+Roughy speaking, we need to:
+
+1. Use boringcrypto version Golang toolset. In TID, we set up golang binaries differently, see [setup_boring_go.sh](https://github.com/istio/istio/blob/f7d03be560753dc71d4c764dceb06f961c4fcdbd/tetrateci/setup_boring_go.sh#L25).
```

Comment:
this should be a link to tetratelabs/istio?

Reply:
good catch. changed.